BGL/include/CGAL/boost/graph/parameters_interface.h

```diff
@@ -47,6 +47,8 @@ CGAL_add_named_parameter(geom_traits_t, geom_traits, geom_traits)
 CGAL_add_named_parameter(vertex_incident_patches_t, vertex_incident_patches, vertex_incident_patches_map)
 CGAL_add_named_parameter(density_control_factor_t, density_control_factor, density_control_factor)
 CGAL_add_named_parameter(use_delaunay_triangulation_t, use_delaunay_triangulation, use_delaunay_triangulation)
+CGAL_add_named_parameter(use_2d_constrained_delaunay_triangulation_t, use_2d_constrained_delaunay_triangulation, use_2d_constrained_delaunay_triangulation)
+CGAL_add_named_parameter(max_squared_distance_t, max_squared_distance, max_squared_distance)
```

Comment:
Since we have `threshold_distance`, what about `threshold_squared_distance()`? If the square is not "important" you could also use the distance.

Reply:
Yes, maybe I should reuse the distance without square. I now think that it is simpler to understand as a parameter name. I will fix that.